init.py

```diff
@@ -422,7 +422,7 @@ def check(reaction, user):
                 # output the collected data to a embed
                 embedVar = discord.Embed(description="", color=0xdab022)  # settings of embed
                 for output_row in output:  # iterate through the prepared list od rewarded items
-                    output_row[0] = output_row[0][2:4] + "/" + output_row[0][0:2]  # prepare date to be DD/MM
+                    output_row[0] = output_row[0][-2:] + "/" + output_row[0][4] + output_row[0][5] + "/" + output_row[0][:4]   # prepare date to be DD/MM/YY
```

Comment:
for some reason i was failing to get the 4th and 5th index for the year so i did them seperately lol...? not sure why `output_row[0][4:5]` didn't work